bin/y-build

```diff
@@ -10,7 +10,10 @@ BUILDCTL_OPTS="$@"
 
 # Settings
 REGISTRY=builds-registry.ystack.svc.cluster.local
-BUILDKIT_CACHE="type=registry,ref=$REGISTRY/ystack:buildcache"
+[ -z "$BUILDKIT_CACHE" ] && BUILDKIT_CACHE="type=registry,ref=$REGISTRY/ystack:buildcache"
+[ -z "$IMPORT_CACHE" ] && IMPORT_CACHE="--import-cache=$BUILDKIT_CACHE"
```

Comment:
Does this mean that I can run `IMPORT_CACHE=false y-build` or something like that in order to disable the cache flag?

Reply:
If not please let me know how you meant to disable the flag :-D

Reply:
> See e4e55f0 for how.